dev-utils/poly-cli/index.js

```diff
@@ -91,10 +91,9 @@ function interactiveSetup() {
             } else if (answers.type === "importer") {
                 handleCreateImporterFeature();
             } else {
-                printErrorMsg(
+                throw Error(
```

Comment:
As long as we're not printing directly from the function, we could refactor these functions to return the formatted string, instead of printing it. The format adds nice visual information for that.

Reply:
Also, every case where we use printErrorMsg is now a candidate for throwing an exception, so we can in that case just change every instance here.

Reply:
I have reworked the entire error handling mechanism. Errors are caught at top level (the functions that we pass to yargs). I think that works quite well.